lib/listener-tracker.js

```diff
@@ -203,12 +226,21 @@ SubscriptionTracker.prototype = {
 
 exports.wrap = function(targetEventEmitter) {
     var wrapper = new EventEmitterWrapper(targetEventEmitter);
-    targetEventEmitter.once('destroy', function() {
+    if (targetEventEmitter.once) {
+      targetEventEmitter.once('destroy', function() {
+          wrapper._listeners.length = 0;
+      });
+    } else {
+      var listener = function() {
         wrapper._listeners.length = 0;
-    });
+        targetEventEmitter.removeEventListener('DOMNodeRemovedFromDocument', listener);
```

Comment:
this is another tricky point, we had to use the `once` work around. But also, how can we accurately listen for when a dom node is removed (window and document we don't have to worry about)? From the specs all the dom events for this are deprecated, but the newer option (mutation observers) are only available in newer browsers.. 

Maybe this is an edge case? I'm open to thoughts/ideas on this one.